fetch/api/basic/stream-safe-creation.any.js

```diff
@@ -0,0 +1,33 @@
+// These tests verify that stream creation is not affected by changes to
+// Object.prototype.
+
+const creationCases = {
+  fetch: async () => fetch(location.href),
+  request: () => new Request(location.href),
+  response: () => new Response('hi'),
+  consumeEmpty: () => new Response().text(),
+  consumeNonEmpty: () => new Response(new Uint8Array([64])).text()
+};
+
+for (creationCase of Object.keys(creationCases)) {
+  promise_test(async t => {
+    Object.prototype.start = () => {
+      throw Error('Object.prototype.start was called');
+    };
+    t.add_cleanup(() => delete Object.prototype.start);
+    await creationCases[creationCase];
```

Comment:
Should this and the same line below be `creationCases[creationCase]()`?

Reply:
This is an important problem! Fixed.